### ISSUE TITLE
paper or technical documents

### ISSUE THREAD

**Issue by Owen1234560** [Jakel21/vehicle-ReID-baseline#2]
Could you provide the paper or technical documents of this code. I can't understand the whole  model framework. Thank you!

**Comment by Jakel21** [comment]
> Could you provide the paper or technical documents of this code. I can't understand the whole model framework. Thank you!

Hi, owen! I am sorry that there is no paper for this code. I just apply my experience in pedestrian ReID to vehicle ReID. The frame of this code is simple: the images are input into resnet50 and train with triplet loss(or cross entropy loss). This is a simple baseline and i am still working on vehicle ReID. The latest paper and code will be released when i finish them. You can search 'deep-person-reid' in github to understand this code(i toke this project for reference).

**Comment by Owen1234560** [comment]
Thank you for your prompt reply.
